src/rhsm/m2cryptossl.py

```diff
@@ -0,0 +1,64 @@
+# A compatibility wrapper that adapts m2crypto to the subset of standard lib ssl used in python-rhsm.
```

Comment:
Sorry, this is really minor, but could you move this comment down below the license?  I have trained my brain to just completely ignore the large comment block at the top of every source file and I'm afraid this information will just end up not getting seen by developers.